### PR TITLE
test(@wallet_settings): add account from wallet settings view

### PR DESCRIPTION
**Pull request by anastasiyaig**
https://ethstatus.testrail.net/index.php?/cases/edit/703598/1

### CODE REVIEW

**Comment by anastasiyaig** [comment]
@Valentina1133 @StateOf-Vlado there is a problem with function 

```
    @allure.step('Open wallet settings')
    def open_wallet_settings(self):
        self._open_settings(4)
        return WalletSettingsView()
```
lines 49-52 of `gui/screens/settings.py` -> this function opens Settings - Back Up seed phrase. 
I tried to fix the problem myself but could not build a reliable locator for each ListItem (squish does not see them). I am seeing that now the logic to find an item is based on indices which looks fragile at the first sight
I also noticed that each of the items in the list has its own object name, perhaps we can use that? ( again, i tried but failed :D )

Any help here is more than welcome, thank u !

Screenshot 2023-09-22 at 18 52 04

File: gui/objects_map/settings_names.py [hunk 1]
```diff
@@ -11,6 +11,7 @@
 mainWindow_scrollView_StatusScrollView = {"container": mainWindow_LeftTabView, "id": "scrollView", "type": "StatusScrollView", "unnamed": 1, "visible": True}
 scrollView_MenuItem_StatusNavigationListItem = {"container": mainWindow_scrollView_StatusScrollView, "type": "StatusNavigationListItem", "visible": True}
 
+
```
Comment (StateOf-Vlado) [inline gui/objects_map/settings_names.py:14]: remove it please

Comment (anastasiyaig) [reply gui/objects_map/settings_names.py:14]: sure!

File: gui/screens/settings.py [hunk 2]
```diff
@@ -110,7 +111,8 @@ def bio(self, value: str):
     def social_links(self) -> dict:
         self._scroll_view.vertical_scroll_to(self._add_more_links_label)
         links = {}
-        for link_name in walk_children(driver.waitForObjectExists(self._links_list.real_name, configs.timeouts.UI_LOAD_TIMEOUT_MSEC)):
+        for link_name in walk_children(
+                driver.waitForObjectExists(self._links_list.real_name, configs.timeouts.UI_LOAD_TIMEOUT_MSEC)):
```
Comment (StateOf-Vlado) [inline gui/screens/settings.py:115]: self.links_list.object doesn't work here?

Comment (anastasiyaig) [reply gui/screens/settings.py:115]: i have not changed anything there, i just fixed the formatting the way PyCharm suggested , that's all @StateOf-Vlado

**Review by StateOf-Vlado — APPROVED** [review]
Perfect, I left one comment, check if it works for you

File: tests/test_wallet/test_wallet_settings.py [hunk 3]
```diff
@@ -149,3 +152,39 @@ def test_change_account_order_not_possible(main_screen: MainWindow, default_name
...
+@pytest.mark.parametrize('name, color, emoji, emoji_unicode, '
+                         'new_name, new_color, new_emoji, new_emoji_unicode', [
+                             pytest.param('GenAcc1', '#2a4af5', 'sunglasses', '1f60e',
+                                          'GenAcc1edited', '#216266', 'thumbsup', '1f44d')
+                         ])
+def test_add_new_account_from_wallet_settings(main_screen: MainWindow, user_account,
+                                  color: str, emoji: str, emoji_unicode: str,
+                                  name: str, new_name: str, new_color: str, new_emoji: str, new_emoji_unicode: str):
+
+    with (step('Open add account pop up from wallet settings')):
```
Comment (Valentina1133) [inline tests/test_wallet/test_wallet_settings.py:170]: Looks like extra brackets were added

**Review by Valentina1133 — APPROVED** [review]

File: tests/test_wallet/test_wallet_settings.py (hunk 3 above)
Comment (anastasiyaig) [reply tests/test_wallet/test_wallet_settings.py:170]: riiiight, thank you!

**Pull request merged**